docs/contributers.rst

```diff
@@ -21,3 +21,14 @@ Principles on the application evolution:
 7. More configuration parameters do not always mean more flexibility. We do not want the application to turn into configuration hell which nobody can use.
 8. The idea of community members is very important because this repo is just a sequence of statements if nobody uses and benefit.
 9. A change will be accepted if it will benefit the community. Particular requests which only very few people will take advantage will not be accepted to leave the code as simple as possible. 
+
+
```

Comment:
Add a disclaimer here stating something like:
"this project started in the spirit of free and open source software with voluntary contributions without compensation. You can always donate your contribution free of charge"